llvm/lib/CodeGen/MIRParser/MIParser.cpp

```diff
@@ -1949,6 +1956,20 @@ bool MIParser::parseLowLevelType(StringRef::iterator Loc, LLT &Ty) {
     Ty = LLT::pointer(AS, DL.getPointerSizeInBits(AS));
     lex();
     return false;
+  } else if (Token.range().front() == 'f') {
+    auto ScalarSize = APSInt(Token.range().drop_front()).getZExtValue();
```

Comment:
can you use consume_front? These also need parsing tests, particularly for the error cases